scrapers/freeones/site.py

```diff
@@ -0,0 +1,241 @@
+from scrapers.scraper import Scraper
+from scrapers.freeones.album import FreeOnesAlbum
+from scrapers.freeones.videos import FreeOnesVideo
+import requests
+import json
+import yaml
+import re
+from lxml import etree
+from bs4 import BeautifulSoup
+from os import makedirs
+
+
+class FreeOnes(Scraper):
+    base_url = "https://www.freeones.com"
+    out_path = f"./babes"
+    album_page = 0
+    videos_page = 0
+    months = {
+        'jan': 1,
+        'feb': 2,
+        'mar': 3,
+        'apr': 4,
+        'may': 5,
+        'jun': 6,
+        'jul': 7,
+        'aug': 8,
+        'sep': 9,
+        'oct': 10,
+        'nov': 11,
+        'dec': 12
+    }
+
+    def __init__(self, write_log):
+        super().__init__(write_log)
+
+        self.babe_path = self.out_path
+        self.album_list = []
+        self.video_list=[]
+        self.log = write_log
+
+        if self.log:
+            print(f"FreeOnes scraper")
+
+    def list_targets(self, page=1):
+        babes = []
+        list_url = f"{self.base_url}/babes?s=rank.currentRank&o=asc&l=12&p={page}"
+        page = requests.get(list_url)
+        soup = BeautifulSoup(page.content, "html.parser")
+        babes_boxes = soup.findAll("div", {'class': ['grid-item', 'teaser-subject']})
+        for box in babes_boxes:
+            feed_link = box.find('a', class_="teaser__link")['href']
+            feed_parts = feed_link.split('/')
+            babes.append(feed_parts[1])
+
+        return babes
+
+    def select_target(self, name):
+        super().select_target(name)
+        self.babe_path = f"{self.out_path}/{self.target}/"
+
+    def bio(self):
+        if self.biography is not None:
+            return self.biography
+
+        self.biography = {}
+        try:
+            with open(r"./CommunityScrapers/scrapers/FreeonesCommunity.yml") as file:
+                structure = yaml.load(file, Loader=yaml.FullLoader)
+            bio_paths = structure['xPathScrapers']['performerScraper']['performer']
+        except IOError as ex:
+            if self.log:
+                print("Structure YAML file not found")
```

Comment:
modify the pull request to introduce a git submodule dependency on the upstream yaml file from stash

Reply:
The dependency is added via the .gitmodules file. Do I miss something? The YAML file is read from the submodule source